changelog.txt

```diff
@@ -1,3 +1,14 @@
+0.1.5-preview
+============
+### New Features:
+**Microsoft Identity Web supports certificates.** The developer can now use client and token decryption certificates, which can be retrieved from a variety of sources, like Azure Key Vault, certificate store, a Base54 encoded string, and more. The location of the certificate can be specified in a configuration file or programmatically. See [issue](https://github.com/AzureAD/microsoft-identity-web/issues/165) and [wiki](https://github.com/AzureAD/microsoft-identity-web/wiki/Using-certificates) for more details.
+**Microsoft Identity Web allows to send client certificates as an `x5c` claim when acquiring tokens.** To enable this behavior set the `SendX5C` property in configuration file. See [issue](https://github.com/AzureAD/microsoft-identity-web/issues/197) for more details.
+**Microsoft Identity Web provides an option to force redirect URIs to use the HTTPS scheme,** which can be useful in certain scenarios, like app deployment in a container. To enable this behavior set `ForceHttpsRedirectUris` property in the configuration file. See [issue](https://github.com/AzureAD/microsoft-identity-web/issues/175) for more details.
+**Microsoft Identity Web uses `System.Text.Json` namespace instead of `Newtonsoft.Json` for working with JSON.** See [issue](https://github.com/AzureAD/microsoft-identity-web/issues/182) for more details.
```

Comment:
is this a feature or a bug fix?

Reply:
Hmm the issue is tagged as a bug, but nothing was really broken.

Reply:
it's a bug because we can't use newtonsoft in net core 5

Reply:
Ok. Moved to bug fixes.